### PR TITLE
LPS-88814 Add SelectField client tests

### PR DESCRIPTION
**Pull request by mclomello**

### CODE REVIEW

File: modules/apps/headless-apio/forms/forms-apio-client-test/src/testIntegration/java/com/liferay/forms/apio/client/test/internal/activator/SelectFieldFormApioTestBundleActivator.java [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+package com.liferay.forms.apio.client.test.internal.activator;
+
+import com.liferay.dynamic.data.mapping.annotations.DDMForm;
+import com.liferay.dynamic.data.mapping.annotations.DDMFormField;
+
+/**
+ * @author Marcelo Mello
+ */
+public class SelectFieldFormApioTestBundleActivator
```
Comment (phcp) [inline modules/apps/headless-apio/forms/forms-apio-client-test/src/testIntegration/java/com/liferay/forms/apio/client/test/internal/activator/SelectFieldFormApioTestBundleActivator.java]: Rename to SelectFormApioTestBundleActivator

File: modules/apps/headless-apio/forms/forms-apio-client-test/src/testIntegration/java/com/liferay/forms/apio/client/test/SelectFormApioTest.java [hunk 2]
```diff
@@ -0,0 +1,266 @@
...
+		).statusCode(
+			200
+		).root(
+			"_embedded.formPages._embedded[0]._embedded.fields._embedded." +
+				"find {it.name == '%s'}",
+			withArgs(
+				SelectFieldFormApioTestBundleActivator.
+					MULTIPLE_SELECT_FIELD_NAME)
+		).body(
+			"multiple", equalTo(true)
```
Comment (phcp) [inline modules/apps/headless-apio/forms/forms-apio-client-test/src/testIntegration/java/com/liferay/forms/apio/client/test/SelectFormApioTest.java:215]: Please check the other field which has the property multiple equals false.

**Review by phcp — CHANGES_REQUESTED** [review]

File: modules/apps/headless-apio/forms/forms-apio-client-test/src/testIntegration/java/com/liferay/forms/apio/client/test/internal/activator/SelectFieldFormApioTestBundleActivator.java (hunk 1 above)
Comment (mclomello) [reply modules/apps/headless-apio/forms/forms-apio-client-test/src/testIntegration/java/com/liferay/forms/apio/client/test/internal/activator/SelectFieldFormApioTestBundleActivator.java]: Renamed was requested. Thanks.

File: modules/apps/headless-apio/forms/forms-apio-client-test/src/testIntegration/java/com/liferay/forms/apio/client/test/SelectFormApioTest.java (hunk 2 above)
Comment (mclomello) [reply modules/apps/headless-apio/forms/forms-apio-client-test/src/testIntegration/java/com/liferay/forms/apio/client/test/SelectFormApioTest.java:215]: Add check multiple equals false for other field. Ty.

**Review by phcp — APPROVED** [review]

**Pull request merged**